File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/builder/vechicle/Car.java
package com.bigboxcode.designpattern.builder.vechicle;

public class Car {
    private int wheel;
    private int engine;
    private int seat;
    private int door;

    private boolean interior;

    public Car(int noOfWheel, int noOfEngine, int noOfSeat, int noOfDoor, boolean interior) {
        this.wheel = noOfWheel;
        this.engine = noOfEngine;
        this.seat = noOfSeat;
        this.door = noOfDoor;
        this.interior = interior;
    }

    public int getWheel() {
        return wheel;
    }

    public int getEngine() {
        return engine;
    }

    public int getSeat() {
        return seat;
    }

    public int getDoor() {
        return door;
    }

    public boolean isInterior() {
        return interior;
    }

    public String toString() {
        return "Car: Wheel -> " + wheel + " | Engine -> " + engine + " | Seat -> " + seat + " | Door -> " + door + " | Interior -> " + interior;
    }
}

File: design-patterns/php/builder/vehicle/Car.php
<?php
// Car.php

namespace Builder\Vehicle;

class Car {
    public function __construct(
        private int $wheel,
        private int $engine,
        private int $seat,
        private int $door,
        private bool $interior) {
    }

    public function getWheel(): int {
        return $this->wheel;
    }

    public function getEngine(): int {
        return $this->engine;
    }

    public function getSeat(): int {
        return $this->seat;
    }

    public function getDoor(): int {
        return $this->door;
    }

    public function isInterior(): bool {
        return $this->interior;
    }

    public function __toString(): string {
        return "Car: Wheel -> " . $this->wheel . " | Engine -> " . $this->engine . " | Seat -> " . $this->seat . " | Door -> " . $this->door . " | Interior -> " . ($this->interior ? 'true' : 'false');
    }
}